packages/react/examples/codesandbox/masthead/src/App.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
Hey @asudoh , looks good! For the examples, is the idea that this doubles as the example codesandbox for adopters?

cc: @kennylam @RaphaelAmadeu

Reply:
Thanks @jeffchew, you are right that the example CodeSandbox can serve for documentation purpose, too.

Reply:
For codesandbox I think we'd want the async call in place rather than bypassing with static data, I guess maybe we can have one example with static data that's used for integration testing, and another default one that uses the service call?

Reply:
@jeffchew Updated the CSB example not to use the mock data. And I may have been wrong that CSB code hosted on GH cannot be configured with environment variable... I could do it for my fork with my GH account. Let's try with our account once we get this PR in master.